edm.yaml

```diff
@@ -9,41 +9,41 @@ components:
 # Components cannot be stored in a collection
 
   fcc::LorentzVector:
-    Mass : float
-    Px : float
-    Py : float
-    Pz : float
+    mass : float
+    px : float
+    py : float
+    pz : float
 
   fcc::Point:
-    X : float
-    Y : float
-    Z : float
+    x : float
+    y : float
+    z : float
 
   # really need to expose some function f(Cellid) -> position
   fcc::BareHit:
-    Cellid : ulonglong
-    Energy : float
-    Time : float
-    Bits : unsigned
+    cellId : ulonglong
+    energy : float
+    time : float
+    bits : unsigned
 
   fcc::BareCluster:
-    Energy : float
-    Time : float
-    Position : fcc::Point
-    Bits : unsigned
+    energy : float
+    time : float
+    position : fcc::Point
+    bits : unsigned
 
   fcc::BareParticle:
-    Type: int
-    Charge: int
-    Status: unsigned
-    Vertex: fcc::Point
-    P4: fcc::LorentzVector
-    Bits: unsigned
+    type: int
```

Comment:
Benedikt mentioned that `type` is needed in PAPAS and that it is not exactly a `PdgId`.. Should we add another member for the Id or live with the potential ambiguity?